modules/shared/programs/claude/files/skills/analyzing-da-sessions/references/algorithm.md

```
@@ -2,20 +2,73 @@
 이슈 #671 본문 PHASE-EXTENDED 6번째 metric "FULL 후 finding 0건 분석"은 v1 measure list에 포함하지 않는다. 본 문서의 derived statistic 섹션에서 비율로 보고한다.
 
+## run-da 건강 지표 (health_formula_version: 1)
+
+주간 리포트 파이프라인(`modules/nixos/programs/da-weekly-report/files/weekly_report.py`)은
+`analyze.py` sidecar와 별도로 git 기반 건강 지표를 수집한다. 산식 변경 시
+`health_formula_version`을 증가시키고, 주간 리포트에 baseline 단절을 명시한다.
+
+| 지표 | 산식 (v1) |
+|------|-----------|
+| 문서 크기 | `git -C "$REPO_ROOT" ls-tree -r HEAD --name-only -- modules/shared/programs/claude/files/skills/run-da/` 결과 중 `*.md` 파일을 세고, `/evals/` path segment가 있는 파일은 제외한다. 라인 수는 `git show HEAD:<path>` 내용 기준 총 line count 합계다. |
+| drift 수리 커밋 빈도 | `git -C "$REPO_ROOT" log --since=<KST 주 시작> --until=<KST 주 끝> --first-parent main --format='%H%x00%s%x00%B%x1e' -- modules/shared/programs/claude/files/skills/run-da/`를 record 단위로 파싱한다. subject가 `/fix|refactor|chore/i`에 매치하고, subject+body가 `/drift|참조|사본|dangling|동기화|SSOT/i`에 매치하는 commit 수다. `--oneline` 금지 — body 매칭을 위해 `%B`를 포함한다. |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)